### ISSUE TITLE
Most targets ignore -ExceptionInfo

### ISSUE THREAD

**Issue by snow-jlove** [EsOsO/Logging#93]
The WinEventLog and Console targets include logic to print `-ExceptionInfo` from `Write-Log`. For example: https://github.com/EsOsO/Logging/blob/4.2.13/Logging/targets/Console.ps1#L38-L40.

However, no other targets have similar logic: https://github.com/search?q=execinfo+repo%3AEsOsO%2FLogging+path%3A%2FLogging%2Ftargets

Can similar logic be added to all targets?

**Comment by EsOsO** [comment]
You're right. Is due to the target to "present" the log line (think it about MVC). If you have anything working feel free to PR a patch.
